Include/httpClient/httpClient.h

```diff
@@ -484,7 +484,7 @@ STDAPI HCHttpCallRequestSetTimeout(
 /// <br />
 /// If the service returns an HTTP error with a "Retry-After" header, then all future calls to that API 
 /// will immediately fail with the original error without contacting the service until the "Retry-After" 
-/// time has been reached.<br />
+/// time has been reached if HCHttpCallRequestSetRetryCacheId() has been called prior to cache the Retry-After header for fast fail logic.<br />
```

Comment:
starting and ending a sentence with an if clause is sort of weird grammatically.

recommend frontloading both conditions at the start of the sentence for better readability

> if the service returns an HTTP error with a "Retry-After" header and the title had previously called HCHttpCallRequestSetRetryCacheId(), then all future calls to that API will immediately fail with the original error without contacting the service until the "Retry-After" time has been reached.<br />

Reply:
thanks, fixed.  when i write my novel, i'm going to hire you as an editor